pkg/synthesis/createK8sResources.go

```diff
@@ -21,8 +21,7 @@ func createK8sResources(model *AbstractModelSyn, outDir string) error {
 	if err := os.RemoveAll(outDir); err != nil {
 		return err
 	}
-	k8sPolicies := &k8sPolicies{}
-	policies, adminPolicies := k8sPolicies.toNetworkPolicies(model)
+	policies, adminPolicies := toNetworkPolicies(model)
```

Comment:
should return these policy objects also from `NSXToK8sSynthesis()`.

Reply:
fixed